Misc/NEWS.d/next/Library/2025-03-12-14-55-56.gh-issue-93096.HfBYPA.rst

```diff
@@ -0,0 +1,2 @@
+An undocumented ``python -m random --test`` benchmark is moved into
+``random`` suite of pyperformance. Patch by Semyon Moroz.
```

Comment:
Don't announce the benchmark to be moved to pyperformance until it's actually there. I suggest to just say that the feature has been removed.